testutils/core.go

```diff
@@ -60,6 +61,16 @@ func RandomString() string {
 	return newUUID.String()
 }
 
+const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+func RandStringBytes(n int) string {
```

Comment:
we already have RandomString in the project. Can you use that?

Reply:
No, i can't. It return UUID as string and it has a number as a first symbol, i can't use this string as package name.